src/UI/Component/Button/Factory.php

```diff
@@ -209,5 +209,55 @@ public function month($default);
 	 */
 	public function tag($label, $action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The graphical button combines the recognisability of a graphical element
+	 *     with the explicity of a text-label. It has an obtrusive presentation that
+	 *     indicate important actions on the screen and make them hard to overlook.
+	 *
+	 *   composition: >
+	 *     The graphical button consists of an icon or glyph and a (very short) text.
+	 *
+	 *   effect: >
+	 *     Both, icon/glyph and text, form an entity an can only be operated together.
+	 *     The button has a "engaged"-state: When the button is used to toggle the
+	 *     visibility of an element, it stays engaged until the element is hidden again.
+	 *
+	 *   rival:
+	 *     Primary Button: >
+	 *       Primary Buttons indicate the most important action among a collection
+	 *       of actions, e.g. in a tool bar, controls of a form or in a modal. Graphical
+	 *       buttons make it hard to miss the indicated action by occupying space.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *       Since Graphical Buttons are so obtrusive they MUST only be used
```

Comment:
This rule does not set graphical buttons aside from primary buttons. This needs sharpening.
Why could one not use graphical buttons in forms or toolbars?

Reply:
"The primary button indicates **the** most important action on a screen.", so there at least is a distinction.

Any idea for the sharpening? I mean basically we need a rule that says if it is ok to make an action obtrusive, right? It needs to cover the Page Layout usecase for this button in the sidebar and the Page Editor usecase as a placeholder for a page element to be added, correct?